test/benchmarks/netperf/README.md

```diff
@@ -0,0 +1,121 @@
+# Benchmarking Kubernetes Networking Performance
+
+## Objectives:
+A standardized benchmark to measure Kubernetes networking performance on multiple host platforms and network stacks.
+
+## Implementation and Test Methodology
+
+The benchmark can be executed via a single Python script invocation that triggers all the automated testing located in the orchestrator and worker pods as seen below. The test uses a custom docker container that has the go binary and iperf3 and other tools built into it. 
+The orchestrator pod coordinates the worker pods to run tests in serial order for the 4 scenarios described below, at MTUs (MSS tuning for TCP and direct packet size tuning for UDP).
+Using node labels, the Worker Pods 1 and 2 are placed on the same Kubernetes node, and Worker Pod 3 is placed on a different node. The nodes all communicate with the orchestrator pod service using simple golang rpcs and request work items.
+
+![](images/BenchmarkingKubernetesNetworkingPerformance.svg)
```

Comment:
Images are in images/ - but I cannot get github to display them with a relative link. Local Markdown previews work.

The github Markdown embedded images issue seems to be encountered by many folks, I couldn't find a solution.